src/fix/main.c

```diff
@@ -894,7 +1030,7 @@ static void processFile(int input, int output, char const *name, off_t fileSize)
 		// If modifying the file in-place, we only need to edit the header
 		// However, padding may have modified ROM0 (added padding), so don't in that case
 		if (padValue == UNSPECIFIED)
-			rom0Len = 0x150;
+			rom0Len = (cartridgeType & 0xff00) == TPP1 ? 0x154 : 0x150;
```

Comment:
This expression is repeated above to initialize `ssize_t minSize`; it could be factored out.